### DISCUSSION TITLE
Export Custom Commands?

### DISCUSSION THREAD

**Discussion by shanecowherd** [intitni/CopilotForXcode#361] [General]
Is it possible to export custom commands to share with my team?

Finally started using the /run command today, ran into some permission problems, so I used the following Apple script in the custom command and it worked great!

<img width="792" alt="Screenshot 2023-09-20 at 3 45 08 PM" src="https://github.com/intitni/CopilotForXcode/assets/442439/cb1571dd-e8c7-4f0f-83e9-2f7d8798ef6e">

My use case was a react native build script, but the pod install would apply to more people.

```bash
/run osascript -e 'tell app "Terminal" to do script "cd ~/Developer/SampleProject; pod install"'
```

**Comment by intitni** [comment]
Exporting custom commands sounds like a good idea. I will work on it after I finish the big things at hand.

**Comment by intitni** [comment]
Finally, it is implemented in 0.30.0.

I was planning to implement it after refactoring the storage of custom commands but never had the time. Eventually, I chose to just implement it...
